examples/2d/mesh2d_manual.rs

```diff
@@ -298,14 +299,34 @@ pub fn extract_colored_mesh2d(
     mut previous_len: Local<usize>,
     // When extracting, you must use `Extract` to mark the `SystemParam`s
     // which should be taken from the main world.
-    query: Extract<Query<(Entity, &ViewVisibility), With<ColoredMesh2d>>>,
+    query: Extract<
+        Query<(Entity, &ViewVisibility, &GlobalTransform, &Mesh2dHandle), With<ColoredMesh2d>>,
+    >,
+    mut render_mesh_instances: ResMut<RenderMesh2dInstances>,
 ) {
+    render_mesh_instances.clear();
```

Comment:
It shouldn’t clear I think. Rather the system should run after extract_mesh2d that does clear.